core/modules/xrdsvc/StreamBuffer.cc

```diff
@@ -38,17 +38,42 @@ LOG_LOGGER _log = LOG_GET("lsst.qserv.xrdsvc.StreamBuffer");
 }
 
 
+using namespace std;
+
+
 namespace lsst {
 namespace qserv {
 namespace xrdsvc {
 
-std::atomic<size_t> StreamBuffer::_totalBytes(0);
+
+atomic<int64_t> StreamBuffer::_maxTotalBytes{40'000'000'000};
```

Comment:
Why signed integer? anything that it's about sizes has to be unsigned. The best type is `size_t`.

Reply:
I'm doing subtraction, this is never compared to to an actual size, and never used to access a memory addres. Really bad things can happen if unsigned wraps around 0 due to some race conditon, so unless I really need to use unsigned, I avoid it.